<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>15</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Kristen ITC&quot;;</string>
    </property>
    <property name="text">
     <string>Degree ---</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1083</x>
      <y>40</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Kristen ITC&quot;;</string>
    </property>
    <property name="text">
     <string>Time    ---</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>15</y>
      <width>72</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Kristen ITC&quot;;</string>
    </property>
    <property name="text">
     <string>Aowu</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>40</y>
      <width>72</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Kristen ITC&quot;;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Game</string>
    </property>
    <widget class="QMenu" name="menuDegree">
     <property name="title">
      <string>Degree</string>
     </property>
     <addaction name="actionBasic"/>
     <addaction name="actionmedium"/>
     <addaction name="actionHard"/>
    </widget>
    <addaction name="actionStart"/>
    <addaction name="separator"/>
    <addaction name="menuDegree"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionBasic">
   <property name="text">
    <string>Basic</string>
   </property>
  </action>
  <action name="actionmedium">
   <property name="text">
    <string>Medium</string>
   </property>
  </action>
  <action name="actionHard">
   <property name="text">
    <string>Hard</string>
   </property>
  </action>
  <action name="actionTop">
   <property name="text">
    <string>Top</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
